src/core/ext/filters/max_age/max_age_filter.c

```diff
@@ -288,45 +284,37 @@ static grpc_error* init_channel_elem(grpc_exec_ctx* exec_ctx,
   chand->channel_stack = args->channel_stack;
   chand->max_connection_age =
       DEFAULT_MAX_CONNECTION_AGE_MS == INT_MAX
```

Comment:
1. is tricky since `MAX_CONNECTION_AGE_INTEGER_OPTIONS` treats `DEFAULT_MAX_CONNECTION_AGE_MS` as an `int`, which grpc_millis isn't necessarily (it's `intptr_t`)